geode-redis/src/main/java/org/apache/geode/redis/internal/GeodeRedisServer.java

```diff
@@ -437,18 +422,16 @@ private void startGemFire() {
   }
 
   @VisibleForTesting
-  public RegionProvider getRegionCache() {
-    return regionCache;
+  public RegionProvider getRegionProvider() {
```

Comment:
Don't feel the need to respond to this,  but also curious if this change will allow us to eventually get rid of the region provider concept.

Reply:
Most of the RegionProvider is about managing dynamic regions for lists and sorted sets. If we would change those data structures to be more like our RedisSet then I think the RegionProvider might be able to go away. However it still might make sense to have it to give us the single data region. And it might be helpful in the future if we have a region per DB.
The work on this branch has more to do with getting rid of the KeyRegistrar. Once all of our data types are converted over to just use the new DataRegion, then I think we will find that no code uses the KeyRegistrar methods anymore.